prov/util/src/util_mem_monitor.c

```diff
@@ -347,15 +347,7 @@ void ofi_monitor_unsubscribe(struct ofi_mem_monitor *monitor,
 #include <sys/ioctl.h>
 #include <linux/userfaultfd.h>
 
-/* The userfault fd monitor requires for events that could
```

Comment:
This comment is probably worth leaving in for future reference.

Reply:
Let me split that off into its own commit